src/services/account.js

```diff
@@ -29,16 +29,11 @@ class AccountService {
 
         const company = await Company.create({ name: companyName });
 
-        const account = await Account.create({
+        await Account.create({
             email,
-            password,
+            password: (password),
```

Comment:
Why this change?

Reply:
It was an accident

Reply:
@FranciscoCardoso913 this does not seem to have changed, was it a typo or not?